docs/files.md

```diff
@@ -260,18 +260,23 @@ A file can be unlocked by calling `FilesManager.UnLock(string id)` with the ID o
 await client.FilesManager.UnLock(id: "11111");
 ```
 
-Create a Shared Link
+Create or update a Shared Link
 --------------------
 
-A shared link for a file can be generated by calling
+A shared link for a file can be created or updated by calling
 `FilesManager.CreateSharedLinkAsync(string id, BoxSharedLinkRequest sharedLinkRequest, IEnumerable<string> fields = null)`.
 
-<!-- sample put_files_id_shared_link_create -->
+<!-- sample put_files_id shared_link_create -->
```

Comment:
Are you sure that `put_files_id shared_link_create` is right here?

This page https://developer.box.com/reference/put-files-id--add-shared-link/ using this sample label 
`put_files_id add_shared_link`

And this page https://developer.box.com/reference/put-files-id--update-shared-link/ is using
`put_files_id update_shared_link`.

However when we want to add sample code here https://developer.box.com/guides/shared-links/create-or-update/, we should add `put_files_id add_shared_link` as well as in first example.

Reply:
You may be right. I took the labels from the original Jira ticket, but I see that perhaps we should use `put_files_id add_shared_link` instead (or both). I'll try to clarify this.

Reply:
Fixed